liiatools/datasets/social_work_workforce/sample_data.py

```diff
@@ -0,0 +1,213 @@
+import random
+import string
+from datetime import date, datetime, timedelta
+import pytz
+
+from sfdata_stream_parser.events import StartElement as S, EndElement as E, TextNode as T
```

Comment:
I'm a bit cautious about this type of aliasing. While it shortens StartElement to S and reduces the need for typing, it also makes the overall code less readable. I don't mind shortening things, but they still should make sense on their own if someone random picked up the code.  Maybe StartE, EndE, etc would be a middle ground?

Reply:
I'm happy not to shorten it at all if you think that's best. My main reason for doing this was because Kaj did it previously in his CIN sample data generator found here: https://github.com/SocialFinanceDigitalLabs/csdata-tools/blob/main/csdatatools/datasets/cincensus/sample_data.py

Reply:
True. I suppose it's more of a personal preference really. I like to be able to understand what something is without scrolling or looking it up. To me, "S", "E", and "T" lack the obvious meaning that other shortenings like "Start", "End", and "Text" do. Code usually should be self-documenting in this way. There's no real PEP8 style guidelines on how to name aliases that I can find. Other validation libraries such as PyLint require variables to be at LEAST two letters, however. I'll leave it to your discretion. The key thing is making sure you're consistent so that it is easy to decipher and I'd argue this abbreviation isn't consistent with the rest of the code.

There are exceptions to this, for instance "pd" for Pandas, and "np" for NumPy (but both are two characters instead of one).  So, like I said...I'll leave this to your choice, but thought I'd challenge it to make a point. It may be a rare instance where I disagree with Kaj and he says it's fine. In the end it isn't incredibly important considering, but may prompt a discussion about what we do going forward later on as a standard.